system_tests/speech.py

```diff
@@ -79,6 +79,21 @@ def _make_sync_request(self, content=None, source_uri=None,
                                        speech_context=['Google', 'cloud'])
         return result
 
+    def _make_async_request(self, content=None, source_uri=None,
+                            max_alternatives=None):
+        client = speech.Client(use_gax=False)
```

Comment:
I think it already does.
```
[testenv:system-tests]
basepython =
    python2.7
commands =
    python {toxinidir}/system_tests/attempt_system_tests.py {posargs}
deps =
    {[testing]deps}
passenv = GOOGLE_* TRAVIS* encrypted_*
```

I guess my question is more how would they get run?